README.md

```diff
@@ -35,41 +35,27 @@ This is OperationCode.org's repository for their new front-end implementation us
 
 ## Install
 
-- `npm start` - This begins the development server.
+### Development Build
+- `npm install -g yarn` - Install Yarn Globally
```

Comment:
I was planning on addressing this with the getting started section #32.  I just wanted to get this up there until I get that finished.